lib/spack/spack/test/reporters.py

```diff
@@ -136,9 +136,9 @@ def test_reporters_extract_empty_and_xfail():
 
     parts = spack.reporters.extract.extract_test_parts("fake", outputs)
 
-    assert len(parts) == 2
+    assert len(parts) == 1
```

Comment:
(Question) Why was this test expecting 2 parts before and only 1 now? Apologies, but that is not immediately clear to me.

Reply:
Detected a bug.  Thanks for calling my attention to it as I found a missing 'continue'.